src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -206,6 +212,11 @@ public function getAllPrevious()
         return $exceptions;
     }
 
+    /**
+     * Returns exception trace
+     *
+     * @return Trace
```

Comment:
returning an object instead of an array is a BC break:

- it does not pass the same typehints
- it does not behave the same regarding to mutations